Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs

```diff
@@ -488,6 +488,11 @@ private void EnsureMixedRealityRequirements()
             // We'll enforce that here, then tracking can update it to the appropriate position later.
             CameraCache.Main.transform.position = Vector3.zero;
 
+            // This will create the playspace
+            Transform playspace = MixedRealityPlayspace.Transform;
+            // Parent the camera under the playspace
+            CameraCache.Main.transform.parent = playspace;
```

Comment:
Looks like this should already happen in the `Transform` call. Were you hitting cases where it wasn't?

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit/Utilities/MixedRealityPlayspace.cs#L35